src/save_pokegear.c

```diff
@@ -305,83 +305,83 @@ BOOL sub_0202F1B4(MomsSavings *momsSavings, u8 idx) {
     return (momsSavings->unk_144 >> idx) & 1;
 }
 
-void sub_0202F1C4(MomsSavings *momsSavings, u16 a1, u16 a2) {
+void MomsSavings_GiftQueuePut(MomsSavings *momsSavings, u16 itemID, u16 quantity) {
     int i;
 
     for (i = 0; i < 5; i++) {
-        if (momsSavings->unk_12C[i][1] == 0) {
-            momsSavings->unk_12C[i][0] = a1;
-            momsSavings->unk_12C[i][1] = a2;
+        if (momsSavings->momGiftQueue[i][1] == 0) {
```

Comment:
Style:
To improve legibility, we have two options, either:
a) make the data be stored as a struct (if it matches)
b) have constants defined as ``MOMGIFTQUEUE_ITEMID = 0`` and ``MOMQIFTQUEUE_QUANTITY = 1``